private-class-data/README.md

```diff
@@ -13,6 +13,109 @@ Private Class Data design pattern seeks to reduce exposure of
 attributes by limiting their visibility. It reduces the number of class
 attributes by encapsulating them in single Data object.
 
+## Explanation
+
+Real world example
+
+> Imagine you are cooking a stew for your family for dinner. You want to prevent your family members from comsuming the stew by tasting it while you are cooking, otherwise there will be no more stew for dinner later.
```

Comment:
```suggestion
> Imagine you are cooking a stew for your family for dinner. You want to prevent your family members from consuming the stew by tasting it while you are cooking, otherwise there will be no more stew for dinner later.
```